src/elements/Header/HeaderSubheader.js

```diff
@@ -1,33 +1,43 @@
-import React, { Component, PropTypes } from 'react'
-import classNames from 'classnames'
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
 import META from '../../utils/Meta'
-import { getUnhandledProps } from '../../utils/propUtils'
-
-export default class HeaderSubheader extends Component {
-  static propTypes = {
-    children: PropTypes.node,
-    className: PropTypes.string,
-  }
-  static _meta = {
-    library: META.library.semanticUI,
-    name: 'HeaderSubheader',
-    parent: 'Header',
-    type: META.type.element,
-  }
-  render() {
-    const classes = classNames(
-      'sd-header-subheader',
-      'sub',
-      this.props.className,
-      'header',
-    )
-
-    const props = getUnhandledProps(HeaderSubheader, this.props)
-
-    return (
-      <div {...props} className={classes}>
-        {this.props.children}
-      </div>
-    )
-  }
+import {
+  getUnhandledProps,
+} from '../../utils/propUtils'
+
+function HeaderSubheader(props) {
+  const {
+    children, className,
+  } = props
+
+  const classes = cx('sd-header-subheader', 'sub',
+    className,
+    'header'
+  )
+
+  const SubHeaderComponent = 'div'
+  const rest = getUnhandledProps(HeaderSubheader, props)
+
+  return (
+    <SubHeaderComponent className={classes} {...rest}>
```

Comment:
Done

> Levi Thomason <mailto:notifications@github.com>
> 24 June 2016 at 17:34
>
> In src/elements/Header/HeaderSubheader.js 
> <https://github.com/TechnologyAdvice/stardust/pull/278#discussion_r68424700>:
>
> > +
> > +function HeaderSubheader(props) {
> > + const {
> > + children, className,
> > + } = props
> > +
> > + const classes = cx('sd-header-subheader', 'sub',
> > + className,
> > + 'header'
> > + )
> > +
> > + const SubHeaderComponent = 'div'
> > + const rest = getUnhandledProps(HeaderSubheader, props)
> > +
> > + return (
> > + <SubHeaderComponent className={classes} {...rest}>
>
> Since the component never changes, let's inline the div here:
>
> <div ...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/TechnologyAdvice/stardust/pull/278/files/4317290b6fd2e4980cfd74e736287438f69d4c6c#r68424700>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AAWHs7_dvxGO4qHx5pDEs0ZBC30NYrf5ks5qPAcggaJpZM4I9saB>.
>

-- 
Sent from Postbox 
<https://www.postbox-inc.com/?utm_source=email&utm_medium=siglink&utm_campaign=reach>